scripts/measure-enzyme-to-rtl-tests-ratio/measure-enzyme-to-rtl-tests-ratio.js

```diff
@@ -0,0 +1,82 @@
+const { spawn } = require('child_process');
+
+const API_URL = 'https://sheetdb.io/api/v1/g3uyrf24ruo1v';
+const ENZYME_COLUMN = 'enzyme';
+const RTL_COLUMN = 'rtl';
+const ID_PATTERN = 'id-';
+const NOT_VALID_ID_PATTERN = 'id';
+
+async function getColumnData(columnName) {
+    const columnData = await fetch(API_URL)
+        .then((response) => {
+            return response.json();
+        })
+        .then((data) => {
+            return data;
+        })
+        .catch((error) => {
+            console.error('SheetDB data fetch error: ', error);
+        });
```

Comment:
Two different styles of hadeling async code are mixed here: `async/await` and `Promises`.

If `async/await` is chosen, then we shouldn't use `.then() / .catch()`. The code should look like this (not tested):

```ts
async function fetchColumnData(columnName) {
    try {
        const columnDataResponse = await fetch(API_URL);
        return await columnDataResponse.json();
    } catch (error) {
        console.error('SheetDB data fetch error: ', error);
        return null;
    }
}


async function getColumnData(columnName) {
    const columnData = await fetchColumnData(columnName);
    if (columnData === null) {
        // handle the error
    }
    for (let i = 0; i < columnData.length; i += 1) {
        const currentColumnData = columnData[i];
        if (columnName === ENZYME_COLUMN && currentColumnData.enzyme.includes(ID_PATTERN)) {
            return currentColumnData.enzyme;
        }
        if (columnName === RTL_COLUMN && currentColumnData.rtl.includes(ID_PATTERN)) {
            return currentColumnData.rtl;
        }
    }
    return NOT_VALID_ID_PATTERN;
}
```